src/lib/Controls.js

```diff
@@ -44,10 +44,10 @@ class Controls {
         this.controlsEl.addEventListener('mouseleave', this.mouseleaveHandler);
         this.controlsEl.addEventListener('focusin', this.focusinHandler);
         this.controlsEl.addEventListener('focusout', this.focusoutHandler);
+        this.controlsEl.addEventListener('click', this.clickHandler);
```

Comment:
Should we move the associated `removeEventListener` call out of its conditional check as well?

Reply:
Also, this component is used in all of the viewers across Preview, so this is a wide-ranging change. Does it have any side effects? Have we tested all the various viewers?